packages/syft/setup.cfg

```diff
@@ -86,18 +86,19 @@ data_science =
     evaluate==0.4.2
     recordlinkage==0.16
     # backend.dockerfile installs torch separately, so update the version over there as well!
-    torch==2.3.0
+    torch==2.3.1
 
 dev =
     %(test_plugins)s
     %(telemetry)s
     bandit==1.7.8
-    ruff==0.4.7
+    ruff==0.4.8
     importlib-metadata==7.1.0
     isort==5.13.2
     mypy==1.10.0
     pre-commit==3.7.1
-    safety>=2.4.0b2
+    safety>=2.4.0b2 # must be upgraded to version 3
+    pip-audit==2.7.3
```

Comment:
Likewise let's not add `pip-audit` here.

Reply:
reverted